resources/sdk/purecloudjavascript/templates/HttpRequestOptions.mustache

```diff
@@ -0,0 +1,53 @@
+export class RequestOptions {
```

Comment:
Add headers.
Possibly add constructor to replace the buildRequestOptions method.

Reply:
done